packages/browser-agent/README.md

```diff
@@ -1,27 +1,148 @@
+
+or [yarn](https://yarnpkg.com/)
+
+```bash
+yarn add @newrelic/browser-agent
+```
+
+### Directly in HTML/JS
+
+Using [unpkg](https://unpkg.com/)
+
+```html
+<script src="https://unpkg.com/@newrelic/browser-agent/bundled"></script>
 ```
-import nr from 'nr-browser-core'
 
-nr.setConfiguration({
-  jserrors: {
-    harvestTimeSeconds: 30
-  }
+## Usage Examples
+
+### Basic Setup
+
+```javascript
+<< App.js >>
+// initialize the agent as close to 
+// the top level of the application as possible
+import NR from '@newrelic/browser-agent'
+const options = {
+    // See 'Configuring your application'
+}
+NR.start(options).then(() => {
+    console.log("Browser Agent Initialized!")
 })
+```
+
+### Instrumenting a Micro Front End
 
-nr.init({
-  beaconUrl: 'localhost:8181',
-  licenseKey: 'abc123',
-  appId: 2
+The New Relic Browser Agent can maintain separate configuration scopes for each generated bundle.  Multiple bundles, like those used in certain micro front end patterns, can each report their own scoped data to separate New Relic applications.
+
+```javascript
+// <<< MICRO FRONT END APP 1 >>>
+// ---- App.js ----
+import NR from '@newrelic/browser-agent'
+const options = {
+    // see 'Configuring your application'
+}
+NR.features.JSERRORS.enabled = true
+NR.start(options).then(() => {
+    console.log("Browser Agent Initialized!")
 })
+
+// ---- MyComponent.js ----
+class MyComponent() {
+    try {
+        ...
+    } catch(err) {
+        NR.noticeError(err)
+        // reports to applicationID 1
+    }
+}
 ```
 
-### Future work / ideas
+## Configuring your application
+
+The NR interface's `start` method accepts an `options` object to configure the agent:
+
+```js
+const options = {
+  licenseKey: String
+  applicationID: String
+  beacon: String
+}
+NR.start(options)
+```
+
+### Get application ID, license key, beacon
+
+You can find `licenseKey`, `applicationID` and `beacon` values in the New Relic UI's Browser Application **Settings** page ([one.newrelic.com](https://one.newrelic.com) > Browser > (select an app) > Settings > Application settings.)
+
+![configuration](https://user-images.githubusercontent.com/4779220/169617807-110f3938-8af9-4aa8-b651-7712589b0792.jpg)
+
+## Features
+
+|Feature|Subfeature|Default|Description|
+|-|-|-|-|
+|JSERRORS|enabled |true|Enable's `noticeError` method|
+|JSERRORS|auto |false|Reports all global errors |
+
+
+### JavaScript Errors
+
+This NPM package can currently capture JavaScript Error reporting in two ways. These errors will be reported to the appId & licenseKey specified in your [configuration](#configuring-your-application).
+
+```js
+import NR from '@newrelic/browser-agent'
+
+// enable API scoped to applicationID
+NR.features.JSERRORS.enabled = true
+
+// report global errors to applicationID
+NR.features.JSERRORS.auto = true
+
+// enable features before starting the agent
+NR.start(options)
+```
+
+### Capture JavaScript errors via API
+
+```javascript
+NR.features.jserrors.enabled = true
```

Comment:
It probably needs to be called out somewhere that these values (`enabled` and `auto`) are only used before the `.start(...opts)` command is called.  Changing them after `.start(...opts)` will have no effect.

Reply:
This could be something we evaluate and iterate on in the future for behavior purposes (like shutting down the agent or some of its running features)